emily/handler/src/common/error.rs

```diff
@@ -92,15 +100,60 @@ impl Error {
             Error::Unauthorized => "Unauthorized access - check your API key".to_string(),
             Error::NotFound => "Resource not found".to_string(),
             Error::NotAcceptable => "Not acceptable format requested".to_string(),
-            Error::NotImplemented(path) => format!("Handling {path:?} is not implemented."),
+            Error::NotImplemented => "Not implemented".to_string(),
             Error::Conflict => "Request conflict".to_string(),
             Error::InternalServer => "Internal server error".to_string(),
+            Error::Debug(s) => format!("Debug error: {s}"),
             Error::ServiceUnavailable => "Service unavailable".to_string(),
             Error::RequestTimeout => "Request timeout".to_string(),
         }
     }
 }
 
+/// TODO(TBD): Route errors to the appropriate Emily API error.
+///
+/// Implement from for API Errors.
+impl From<SdkError<GetItemError>> for Error {
+    fn from(err: SdkError<GetItemError>) -> Self {
+        Error::Debug(format!("SdkError<GetItemError> - {err:?}"))
+    }
+}
```

Comment:
I'm not sure what you're advocating for - are you saying that we shouldn't implement "From" for errors or that you don't advise doing what you suggested? 

What would you change about the existing `From` implementations here (beyond having a sane response different from what I've written in).

Reply:
Oh, I'm not advocating for anything. I saw these `From` implementations and thought that just you wanted a fast way to enable the `?` operator in your code and that a quicker way to do the same thing would be something like my suggestion above. This quicker way also brings the code closer to what the "final" code would be. I may have misread what you wanted to do but that was where most of my comment was coming from.

As for this part:
> In general, I do not want to encourage From implementations for error variants but I think it's fine as we build things out.

Well, for error handling I've been encouraging a kinda "context enhancing" approach in the signer codebase. That approach shuns `#[from]` implementations (since you can eschew adding the context). But in the interest of time, we sometimes don't follow it too closely so I don't expect you to follow it too closely. But I don't want to encourage anyone to move away from it either. Also, I didn't want to say all of that so I said something shorter and more opaque.